Add release note for e5.8.1 (#2684)
emqtt/emq-docs-cn: en_US/changes/breaking-changes-ee-5.8.md

# Incompatible Changes in EMQX 5.8

## e5.8.1

> - [#13792](https://github.com/emqx/emqx/pull/13792) The default expiration time for a banned item that is created without an `until` value is now `infinity` (previsouly capped at 1 year limit).

Reviewer: both breaking changes should be added to ce as well

Reviewer 2: done

# Incompatible Changes in EMQX 5.8

## e5.8.1

> - [#13792](https://github.com/emqx/emqx/pull/13792) The `GET /banned` endpoint supports querying using filters in the query string.

Reviewer: In this file we only need to leave the last sentence from the changelog entry:

```
 The default expiration time for a banned item that is created without an `until` value is now `infinity` up from 1 year.
 ```

  The available filters are:
  - clientid
  - username
  - peerhost
  - like_clientid
  - like_username
  - like_peerhost
  - like_peerhost_net
  
  The default expiration time for a banned item that is created without an `until` value is now `infinity` up from 1 year.

> - [#13742](https://github.com/emqx/emqx/pull/13742) Fixed when subscribing with `+` as the first level, or `#` as a wildcard, retained messages with topics starting with `$` are incorrectly received.

Reviewer: ```suggestion
- [#13742](https://github.com/emqx/emqx/pull/13742) Fixed an issue when a client would receive retained messages for topic starting with `$` when it subscribed to topic `#` or `+`.
```
